doc/source/reference/api/pandas.Series.__invert__.rst

```diff
@@ -0,0 +1,53 @@
+.. currentmodule:: pandas
```

Comment:
With the request above, this file can be removed. Add anything necessary to the docstring of `__invert__` instead.